L1Trigger/Phase2L1ParticleFlow/src/newfirmware/jetmet/TauNNIdHW.cc

```diff
@@ -0,0 +1,119 @@
+#include <iostream>
+#include "L1Trigger/Phase2L1ParticleFlow/src/newfirmware/jetmet/TauNNIdHW.h"
+#include "L1Trigger/Phase2L1ParticleFlow/src/newfirmware/jetmet/nnet_layer.h"
+#include "L1Trigger/Phase2L1ParticleFlow/src/newfirmware/jetmet/nnet_activation.h"
```

Comment:
Why all these are in jetmet?

I think the best would be to move:
 - the common part which is not nn tau specific into a new directory `newfirmware/common/hls4ml`
 - the NN tau specific part under `newfirmware/taus/nn`

Reply:
moved to taus, and organized things to more like an sw framework. They were just there b/c that is where I got it to work.